client_api.go

```diff
@@ -68,6 +68,20 @@ type SOAResults struct {
 	Errors []string `json:"errors,omitempty"`
 }
 
+// RevocationReason is a type for specifying the reason why a certificate is being
+// revoked when requesting revocation.
+type RevocationReason string
+
+// Revocation reasons to provide when revoking a certificate and providing a
+// reason for its revocation.
+const (
+	RevocationReasonUnspecified          = "unspecified"
```

Comment:
should we use the RevocationReason("unspecified") type here ? same for all ?

Reply:
Yup!